src/main/webapp/app/exam/participate/exercises/quiz/quiz-exam-submission.component.ts

```diff
@@ -47,7 +47,18 @@ export class QuizExamSubmissionComponent extends ExamSubmissionComponent impleme
     @Input()
     studentSubmission: AbstractQuizSubmission;
 
-    @Input() exercise: QuizExercise;
+    @Input() exerciseId: number | undefined;
+
+    @Input() quizQuestions: QuizQuestion[] | undefined;
+
+    @Input() randomizeQuizQuestions: boolean | undefined;
+
+    @Input() title: string | undefined;
```

Comment:
it's a bit cumbersome to define to pass so many attributes of the quiz. I understand that you need to provide an exercise independent way of passing values. However, can you not simply define an interface or super class which includes all those values so that you only need to pass one input?